### PR TITLE
Mepts 241 imer report fixes

### PR DESCRIPTION
**Pull request by KelvinWanyama**
- Removed obs with null date from the NextAndPrevDates Calculation class.

### CODE REVIEW

File: api/src/main/java/org/openmrs/module/eptsreports/reporting/calculation/dsd/NextAndPrevDatesCalculation.java [hunk 1]
```diff
@@ -62,6 +62,8 @@ public CalculationResultMap evaluate(
       while (iterator.hasNext()) {
         Obs obs = iterator.next();
         if (obs.getVoided()
+            || obs.getValueDate() == null
```
Comment (nthfloor) [inline api/src/main/java/org/openmrs/module/eptsreports/reporting/calculation/dsd/NextAndPrevDatesCalculation.java]: getValueDate & getValueDateTime return the same object - do we need both?

Comment (KelvinWanyama) [reply api/src/main/java/org/openmrs/module/eptsreports/reporting/calculation/dsd/NextAndPrevDatesCalculation.java]: GetValueDateTime has the time component while getValueDate has only the date.

Comment (nthfloor) [reply api/src/main/java/org/openmrs/module/eptsreports/reporting/calculation/dsd/NextAndPrevDatesCalculation.java]: They return the same object, see here: https://github.com/openmrs/openmrs-core/blob/06593ec985c2e2064e13d172159b04f4398ca1ae/api/src/main/java/org/openmrs/Obs.java#L804-L823

Comment (KelvinWanyama) [reply api/src/main/java/org/openmrs/module/eptsreports/reporting/calculation/dsd/NextAndPrevDatesCalculation.java]: You are right @nthfloor We only need one, either return both date and time components.

Comment (KelvinWanyama) [reply api/src/main/java/org/openmrs/module/eptsreports/reporting/calculation/dsd/NextAndPrevDatesCalculation.java]: I will go ahead and make the changes.

**Pull request merged**